okhttp/src/test/java/okhttp3/internal/http2/HpackTest.java

```diff
@@ -183,7 +183,7 @@
   /** Header table backing array is initially 8 long, let's ensure it grows. */
   @Test public void dynamicallyGrowsBeyond64Entries() throws IOException {
     // Lots of headers need more room!
-    hpackReader = new Hpack.Reader(16384, 4096, bytesIn);
+    hpackReader = new Hpack.Reader(bytesIn, 16384, 4096);
```

Comment:
We should start naming params of ints and booleans to avoid confusion in the codebase, I suggest starting with changes that come through these PRs.

Reply:
Can do for Kotlin callsites!